src/Lachain.Crypto/TPKE/TrustedKeyGen.cs

```diff
@@ -27,5 +27,11 @@ public PrivateKey GetPrivKey(int i)
         {
             return new PrivateKey(MclBls12381.EvaluatePolynomial(_coeffs, Fr.FromInt(i + 1)), i);
         }
+        
+        public PublicKey GetVerificationPubKey(int i)
+        {
+            return new PublicKey(G1.Generator * MclBls12381.EvaluatePolynomial(_coeffs, Fr.FromInt(i + 1)), i);
```

Comment:
I think the 2nd input in PublicKey(_ , i) should be _degree, I think it is faulty node count

Reply:
yes,  you are right, copy-paste error.)